ext/Logger.php

```diff
@@ -91,33 +119,40 @@ public function afterTest(TestEvent $e)
 
     public function endTest(TestEvent $e)
     {
-        $this->logger->info("PASSED");
+        self::info("PASSED");
+        self::$logger = self::getDefaultLogger();
     }
 
     public function testFail(FailEvent $e)
     {
-        $this->logger->alert($e->getFail()->getMessage());
-        $this->logger->info("# FAILED #");
+        self::alert($e->getFail()->getMessage());
+        self::info("# FAILED #");
     }
 
     public function testError(FailEvent $e)
     {
-        $this->logger->alert($e->getFail()->getMessage());
-        $this->logger->info("# ERROR #");
+        self::alert($e->getFail()->getMessage());
+        self::info("# ERROR #");
     }
 
     public function testSkipped(FailEvent $e)
     {
-        $this->logger->info("# Skipped #");
+        self::info("# Skipped #");
     }
 
     public function testIncomplete(FailEvent $e)
     {
-        $this->logger->info("# Incomplete #");
+        self::info("# Incomplete #");
     }
 
     public function beforeStep(StepEvent $e)
     {
-        $this->logger->info((string) $e->getStep());
+        self::info((string)$e->getStep());
+    }
+
+    public static function __callStatic($name, $arguments)
```

Comment:
but I'm highly against this `__callStatic` it really breaks SOLID principle. This class is not a logger, it is an extension which wraps logger, so it shouldn't be used this way.